presto-geospatial/src/test/resources/us-states.tsv

```diff
@@ -32,10 +32,10 @@ NM	POLYGON ((-107.421329 37.000263, -106.868158 36.994786, -104.337812 36.994786
 NC	POLYGON ((-80.978661 36.562108, -80.294043 36.545677, -79.510841 36.5402, -75.868676 36.551154, -75.75366 36.151337, -76.032984 36.189676, -76.071322 36.140383, -76.410893 36.080137, -76.460185 36.025367, -76.68474 36.008937, -76.673786 35.937736, -76.399939 35.987029, -76.3616 35.943213, -76.060368 35.992506, -75.961783 35.899398, -75.781044 35.937736, -75.715321 35.696751, -75.775568 35.581735, -75.89606 35.570781, -76.147999 35.324319, -76.482093 35.313365, -76.536862 35.14358, -76.394462 34.973795, -76.279446 34.940933, -76.493047 34.661609, -76.673786 34.694471, -76.991448 34.667086, -77.210526 34.60684, -77.555573 34.415147, -77.82942 34.163208, -77.971821 33.845545, -78.179944 33.916745, -78.541422 33.851022, -79.675149 34.80401, -80.797922 34.820441, -80.781491 34.935456, -80.934845 35.105241, -81.038907 35.044995, -81.044384 35.149057, -82.276696 35.198349, -82.550543 35.160011, -82.764143 35.066903, -83.109191 35.00118, -83.618546 34.984749, -84.319594 34.990226, -84.29221 35.225734, -84.09504 35.247642, -84.018363 35.41195, -83.7719 35.559827, -83.498053 35.565304, -83.251591 35.718659, -82.994175 35.773428, -82.775097 35.997983, -82.638174 36.063706, -82.610789 35.965121, -82.216449 36.156814, -82.03571 36.118475, -81.909741 36.304691, -81.723525 36.353984, -81.679709 36.589492, -80.978661 36.562108))
 ND	POLYGON ((-97.228743 49.000239, -97.097296 48.682577, -97.16302 48.545653, -97.130158 48.140359, -97.053481 47.948667, -96.856311 47.609096, -96.823449 46.968294, -96.785111 46.924479, -96.801542 46.656109, -96.719387 46.437031, -96.598895 46.332969, -96.560556 45.933153, -104.047534 45.944106, -104.042057 47.861036, -104.047534 49.000239, -97.228743 49.000239))
-OH	POLYGON ((-80.518598 41.978802, -80.518598 40.636951, -80.666475 40.582182, -80.595275 40.472643, -80.600752 40.319289, -80.737675 40.078303, -80.830783 39.711348, -81.219646 39.388209, -81.345616 39.344393, -81.455155 39.410117, -81.57017 39.267716, -81.685186 39.273193, -81.811156 39.0815, -81.783771 38.966484, -81.887833 38.873376, -82.03571 39.026731, -82.221926 38.785745, -82.172634 38.632391, -82.293127 38.577622, -82.331465 38.446175, -82.594358 38.424267, -82.731282 38.561191, -82.846298 38.588575, -82.890113 38.758361, -83.032514 38.725499, -83.142052 38.626914, -83.519961 38.703591, -83.678792 38.632391, -83.903347 38.769315, -84.215533 38.807653, -84.231963 38.895284, -84.43461 39.103408, -84.817996 39.103408, -84.801565 40.500028, -84.807042 41.694001, -83.454238 41.732339, -83.065375 41.595416, -82.933929 41.513262, -82.835344 41.589939, -82.616266 41.431108, -82.479343 41.381815, -82.013803 41.513262, -81.739956 41.485877, -81.444201 41.672093, -81.011523 41.852832, -80.518598 41.978802, -80.518598 41.978802))
```

Comment:
These seem like unrelated changes.

Reply:
It is needed, otherwise it fails `throw new PrestoException(INVALID_FUNCTION_ARGUMENT, "Polygon is not valid: it has two identical consecutive vertices");`